src/index.html

```diff
@@ -6,9 +6,41 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" 
+     href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+         src="images/logo.png"
+         alt="Company-logo"
+         width="40"
+         height="40">
```

Comment:
<img width="1073" alt="Pasted Graphic" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/8f649a67-d453-4116-a895-91aa03be30a1">
fix it everywhere

Reply:
In what chat exactly can I ask for help? How is that chat called?